test/oli/accounts_test.exs

```diff
@@ -104,6 +104,63 @@ defmodule Oli.AccountsTest do
 
       assert author.preferences.timezone == "America/Los_Angeles"
     end
+
+    test "update author password successfully" do
```

Comment:
I'm wondering if we could check more that the password here because we are handling different parts of information like: given_name, family_name, email, and passwords that also could be checked. Practically what the user chan change from the edit form.


I think this test would be